sdk/communication/test-resources/test-resources.json

```diff
@@ -58,7 +58,7 @@
     },
     {
       "type": "Microsoft.Authorization/roleAssignments",
-      "apiVersion": "2019-04-01-preview",
+      "apiVersion": "2022-04-01",
```

Comment:
it should be update in other platforms as well?

Reply:
I had already done that and they are merged to main